api_docs/templates/partials/footer.html

```diff
@@ -49,7 +49,7 @@ <h3 class="title">Support</h3>
     </div><!--//row-->
     <div class="has-divider signature">
       <div class="footer-col download col-md-6 col-sm-12 col-xs-12">
-        <small><a href="https://www.botify.com/terms-privacy/">Privacy Policy Terms &amp; Conditions</a> | Botify 2018</small>
+        <small><a href="https://www.botify.com/terms-privacy/">Privacy Policy Terms &amp; Conditions</a> | Botify 2020</small>
```

Comment:
You couldn't know, but the new link for privacy and term is `/privacy-and-terms`

Reply:
it was still redirecting to the right page though.
Anyway, I updated it to be sure

Reply:
right page on the old website, bad page on the new :) Thanks for the update!